annlite/executor.py

```diff
@@ -9,57 +9,47 @@
 from jina import Executor, requests
 from jina.logging.logger import JinaLogger
 
-from .index import AnnLite
-
 
 class AnnLiteIndexer(Executor):
     """
-    A simple indexer that wraps the AnnLite indexer and adds a simple interface for indexing and searching.
+    :param n_dim: Dimensionality of vectors to index
+    :param dim: Deprecated, use n_dim instead
+    :param metric: Distance metric type. Can be 'euclidean', 'inner_product', or 'cosine'
+    :param include_metadata: If True, return the document metadata in response
+    :param limit: Number of results to get for each query document in search
+    :param ef_construction: The construction time/accuracy trade-off
+    :param ef_search: The query time accuracy/speed trade-off
+    :param max_connection: The maximum number of outgoing connections in the
+        graph (the "M" parameter)
+    :param index_access_paths: Default traversal paths on docs
```

Comment:
This still exists in code base, not passed in constructor but used in index

Reply:
`ef_seach` and `max_connection` are used to build `self._index` through `config`.
`index_access_paths` will be specified if `index_traversal_paths` is given in **kwargs. you can refer to `__init()__`.